accord-core/src/test/java/accord/burn/BurnTestConfigurationService.java

```diff
@@ -269,10 +272,15 @@ public synchronized void acknowledgeEpoch(long epoch)
     {
         epochs.acknowledge(epoch);
         Topology topology = getTopologyForEpoch(epoch);
-        Node originator = lookup.apply(node);
+        Node originator = originator();
         topologyUpdates.syncEpoch(originator, epoch - 1, topology.nodes());
     }
 
+    private Node originator()
+    {
+        return lookup.apply(node);
```

Comment:
I don't understand this comment.. this function is used, and not sure the relation with `MessageSink`?

Reply:
`MessageSink` is unused because replaced it with `originator()` and calling `Node.send`[ on line 236](https://github.com/apache/cassandra-accord/pull/38/files/bff2a2b6b1aebbc871be4d0d34ba04acf19454a6#diff-8e19f7549abc451cea111cebf36df3a0bdc0c8b42c4f61193cdf05c7c80fdcdcR236)